src/IssueViz/IssueVisualizationControl/ViewModels/IssueVisualizationViewModel.cs

```diff
@@ -83,6 +84,9 @@ internal sealed class IssueVisualizationViewModel : IIssueVisualizationViewModel
                 selectionService.SelectedLocation);
         }
 
+        public bool HasNonNavigableLocations => CurrentIssue != null && 
```

Comment:
I ended up doing it in the code, it just wasn't working in the xaml. In addition, I talked to Marco about the experience that he wants: should we display the footer when there is a non navigable location in _any_ of the flows, or just the active one. He chose in any flow.